modules/ROOT/pages/remote-ign.adoc

```diff
@@ -35,3 +35,23 @@ ignition:
 ----
 
 NOTE: The certificate authorities listed here are not automatically added to the host filesystem. They are solely used by Ignition itself when fetching over `https`. If you'd like to also install them on the host filesystem, include them as usual under the `storage.files` array.
+
+In some cases, you need to merge a local configuration and one or several remote ones, you can use the `merge` rather than `replace` in a Butane config.
```

Comment:
```suggestion
In some cases, if you need to merge a local configuration and one or several remote ones, you can use the `merge` rather than `replace` in a Butane config.
```